### ISSUE TITLE
GameGuru MAX - Option for Smaller Icons - Request

### ISSUE THREAD

**Issue by ArgentArts** [TheGameCreators/GameGuruRepo#1569]
This is a very minor request, but would it be possible to add a setting that allows the end-user to use smaller icons? The icons along the top are HUGE! I have an ultra-wide screen (3440 x 1440) and they are monsters in size, even on my high-resolution screen. I can't imagine how big these things are on someone's laptop. I like to have as much screen real estate available to me for my level/scene as possible. It would be great to recoup a bit of that screen real estate by allowing an option for smaller icons for that top bar. :)

**Comment by rvanner** [comment]
@ArgentArts I think this is a big issue! Can you send a screen shot so we can see what you see?

**Comment by ArgentArts** [comment]
![icons](https://user-images.githubusercontent.com/36735507/142373233-230d569d-dce0-4ba2-bc08-964f4aa8d705.png)

This is taken full-screen, 3440 x 1440. The icons look like they do in the videos that Lee posts every Wednesday. I just think they are too large and, as a result, the top bar eats up screen real estate. If there could be an option for the end-user to make them smaller, that would help, I think. :)

**Comment by rvanner** [comment]
Not as large as I was expecting!
I will set this to low priority for now, we can review it later.

**Comment by ArgentArts** [comment]
I agree. It's low priority. But, for the record, those top-bar icons are somewhere around 68-70 pixels tall. The ones in the Level Objects panel are less than half that at about 32 pixels. :)
